test/test_torch.py

```diff
@@ -6088,6 +6088,101 @@ def test_pow(self, device):
             torch.pow(m1, 1, out=out)
             self.assertEqual(out, m1)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @onlyOnCPUAndCUDA
+    @dtypes(*list(product(torch.testing.get_all_dtypes(include_bool=False, include_complex=False),
+                          torch.testing.get_all_dtypes(include_bool=False, include_complex=False))))
+    def test_float_power(self, device, dtypes):
+        def to_np(value):
+            if isinstance(value, torch.Tensor):
```

Comment:
This can be codegolfed a little:

```
if isinstance(value, torch.Tensor) and value.dtype == torch.bfloat16:
  return value.to(torch.float).cpu().numpy()
```

Reply:
Addressed！